mobify/bellows
# Implemented event delegation for items inside bellows.

Implemented event delegation instead of binding to individual elements. This will allow dynamically added items to work without rebinding. Added a new `add` method to dynamically add items. Also refactored the CSS classes into their own object, so the code is inline with the selectors object. Added a few comments. 

Status: **Ready to merge**

Reviewers: @kpeatt @tedtate @donnielrt 
## Changes
- changed standard event binding on individual items to delegated event binding on bellows parent
- added `add` method, which allows adding items to bellows
- refactored CSS into object, and renamed properties
- added comments
- added tests
## How to test-drive this PR
- checkout this branch
- `grunt test`
- `grunt serve`, ensure no linting errors, and check the examples - specifically test the default example, and bellows within bellows
